wandb/sdk/data_types/base_types/media.py

```diff
@@ -205,7 +205,7 @@ def to_json(self, run: Union["LocalRun", "LocalArtifact"]) -> dict:
                 assert isinstance(self._path, str)
                 assert isinstance(self._sha256, str)
                 artifact = run  # Checks if the concrete image has already been added to this artifact
-                name = artifact.get_added_local_path_name(self._path)
+                name = artifact.get_added_local_path_name(util.FilePathStr(self._path))
```

Comment:
again seems like it should be just handled in the method and we could just use `str` or `Pathlib` type that you have